packages/client/src/bags/types/putBagPromocodes.types.ts

```diff
@@ -0,0 +1,25 @@
+import type { Bag } from './bag.types';
+import type { Config } from '../..';
+
+export type PromoCodeInformation = {
+  promoCode: string;
+  isValid: boolean;
+  error: {
+    code: string;
+    message: string;
+  };
+};
+
+export type BagPromoCodesInformation = {
+  promoCodesInformation: PromoCodeInformation[];
+};
+
+export type PutBagPromocodesData = {
+  promocodes: string[];
+};
+
+export type PutBagPromocodes = (
+  id: Bag['id'] | null,
```

Comment:
According to the api spec, this should not be null. Can you confirm, please?

Reply:
You are right. Fixed